etc/kubernetes/helm/values.yaml

```diff
@@ -0,0 +1,26 @@
+image: elyra/enterprise-gateway:dev
```

Comment:
The tag should be EG-version specific.  We have a [release script](https://github.com/jupyter/enterprise_gateway/blob/master/release.sh) that replaces `VERSION` with the appropriate value.  I'm also thinking that things like these charts and the k8s eg yaml file should be included in the distribution output somehow.

Reply:
Good catch. Just pushed a change to that release script that sets the version in the Helm chart's `values.yaml` upon release. Testing done: While I wasn't able to run the whole release script (maybe a local virtualenv issue), I did confirm it got through the version substitution part, and successfully replaced the version in the Helm `values.yaml` as expected.

As for distributing the K8s YAML, I'm open to ideas on where to add them! It looks like the Makefile produces several build/release artifacts: Python wheels, Python source dists, Docker images, etc. I'm not sure it makes sense to put the K8s YAML into any of those, so maybe it should be a separate build artifact on its own? Or see below about the source tarball. Honestly, the fact that distributing K8s YAML is kind of a pain is part of the problem that Helm attempts to solve.

As for the Helm chart, the canonical way for distributing charts is to stand up a chart server that the Helm client can pull from. (Analogous to a container image server, but for Helm charts.) I think it can be as simple as a web server with a little bit of metadata in a standard location, or as full-featured as an [actual server](https://chartmuseum.com/). I thought about including that originally, but didn't want to gate this change on standing up something like that for this project.

Maybe the right place for a v0 is to include both the K8s YAML and the Helm chart in the source release tarball? If so, then the good news is the K8s YAML is already in there, and I imagine the Helm chart would be as well upon the next release. If you like, I can update the documentation to instruct users to use the release tarball for both.

Reply:
Yes - I had forgotten about the source tarball.  That's fine for now.

Reply:
Cool. I've just added a mention in the docs that a source release can be used for either a K8s YAML or Helm install.